lib/libutee/tee_api_property.c

```diff
@@ -516,15 +554,21 @@ TEE_Result TEE_GetPropertyAsBinaryBlock(TEE_PropSetHandle propsetOrEnumerator,
 		goto err;
 	}
 
-	val = pv.u.str_val;
-	val_len = strlen(val);
-	size = *valueBufferLen;
-	if (!base64_dec(val, val_len, valueBuffer, &size)) {
```

Comment:
Currently we're base64 encoding hardcoded binary properties of TAs, if I recall right only because it was convenient at some time.